setup.py

```diff
@@ -1,27 +1,64 @@
+# pylint: disable=C0114
+import re
+
+from pkg_resources import get_distribution, DistributionNotFound
 from setuptools import setup, find_packages
 
 long_description = """A library for image augmentation in machine learning experiments, particularly convolutional
 neural networks. Supports the augmentation of images, keypoints/landmarks, bounding boxes, heatmaps and segmentation
 maps in a variety of different ways."""
 
+INSTALL_REQUIRES = [
+    "six",
+    "numpy>=1.15",
+    "scipy",
+    "Pillow",
+    "matplotlib",
+    "scikit-image>=0.14.2",
+    "opencv-python-headless",
+    "imageio",
+    "Shapely",
+]
+
+ALT_INSTALL_REQUIRES = {"opencv-python-headless": "opencv-python"}
```

Comment:
`["opencv-python", "opencv-contrib-python", "opencv-contrib-python-headless"]` here instead of just `"opencv-python"`.